common/sgx/tcbinfo.c

```diff
@@ -19,12 +19,20 @@
     "ConfigurationAndSWHardeningNeeded"
 #define SGX_TCB_STATUS_INVALID "Invalid"
 
+#ifdef OEUTIL_TCB_ALLOW_ANY_ROOT_KEY // allow overrode by oeutil
+#ifdef _WIN32
+const char* _trusted_root_key_pem = NULL;
+#else  // for gcc and clang
+const char* _trusted_root_key_pem __attribute__((weak));
```

Comment:
Consider use `OE_WEAK` so that we can get rid of `#ifdef _WIN32` and `#else` here